ios/PDFLib.mm

```diff
@@ -3,6 +3,7 @@
 
 #include <PDFWriter.h>
 #include <PDFPage.h>
+#include <PDFUsedFont.h>
```

Comment:
@jacargentina Are you sure it is necessary to `#include` this here? It's been awhile since I've messed with this code, but I took a look at the other files that are using `PDFUsedFont` (e.g. [`PDFPageFactory`](https://github.com/Hopding/react-native-pdf-lib/blob/master/ios/PDFPageFactory.h#L21)) and those files do not have a `#include <PDFUsedFont.h>` line.

Reply:
The new `measureText` method needs to reference the type inside PDFUsedFont, named `TextMeasures`, ie:

```        PDFUsedFont::TextMeasures measures = font->CalculateTextDimensions(text.UTF8String, (long)fontSize);
```

Other way, code doesn't build at all.